api/v1/GameSession.php

```diff
@@ -276,12 +277,17 @@ public function CreateGameSessionAndSignal(
         // get the entire session database in order - bare minimum the database is created and config file is put on
         //  its designated spot
         $update = new Update();
-        $result = $update->ReimportAdvanced(
-            $config_file_path,
-            $geoserver_url,
-            $geoserver_username,
-            $geoserver_password
-        );
+        $e = null;
+        try {
+            $update->ReimportAdvanced(
+                $config_file_path,
+                $geoserver_url,
+                $geoserver_username,
+                $geoserver_password
+            );
+        } catch (Throwable $e) {
+            $e = new Exception('Recreate failed', 0, $e);
```

Comment:
throw a new exception but pass it the previous one such that it can be tracked..